bin/docker_start.sh

```diff
@@ -62,4 +62,5 @@ exec uwsgi \
     --enable-threads \
     --processes $uwsgi_processes \
     --threads $uwsgi_threads \
-    --buffer-size=65535
+    --buffer-size=65535 \
+    --http-timeout=300
```

Comment:
can we make this an environment variable?

It should really be an edge case that an HTTP request takes _this_ long and not be the norm.